src/components/RegistrationForm/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Wrapper from '@components/Wrapper'
+import InputWithLabel from '@components/InputWithLabel'
+import { Form } from './styles'
+
+const RegistrationForm = () => {
+  return (
+    <Wrapper mTop="15px">
+      <Form>
+        <InputWithLabel id="firstname" label="Firstname" />
```

Comment:
> First name

Needs a space. The id should also be `firstName` or `first-name`, whatever style you prefer.